File: in-motion-server/Post/IMS.Post.Models/Dto/Outgoing/PostReactionDto.cs
namespace IMS.Post.Models.Dto.Outgoing;

public class PostReactionDto
{
    public string Id { get; set; }
    public PostAuthorDto Author { get; set; }
    
    public string Emoji { get; set; }
    
    public string PostId { get; set; }
    
    public DateTime CreatedAt { get; set; }
}

File: in-motion-android/app/src/main/java/com/inmotion/in_motion_android/data/remote/dto/posts/reactions/PostReactionDto.kt
package com.inmotion.in_motion_android.data.remote.dto.posts.reactions

import com.inmotion.in_motion_android.data.remote.dto.posts.PostAuthorDto

data class PostReactionDto(
    val id: String,
    val author: PostAuthorDto,
    val emoji: String,
    val postId: String,
    val createdAt: String
)
